CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Added
+
+-   Add option to use Breeze Dark only if the client's dark mode is enabled
```

Comment:
Could you add the following to the changelog line, like it's done on previous entries

`[#98](https://github.com/mwalbeck/nextcloud-breeze-dark/issues/98)`